PA_19.py

```diff
@@ -0,0 +1,60 @@
+#Recursive Solution
+#Time Complexity - Exponential

```

Comment:
Be specific about time complexities, for now, 2 power n, since you are making 2 choices at every point